src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-details/osd-details.component.spec.ts

```diff
@@ -1,9 +1,13 @@
 import { HttpClientModule } from '@angular/common/http';
+import { DebugElement } from '@angular/core';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
+import { of } from 'rxjs';
+
```

Comment:
You can remove this line.

Reply:
When I do that and run `npm run test` I get the following error:

```
  ● OsdDetailsComponent › should succeed creating a histogram

    ReferenceError: of is not defined

      69 |       histogram: {}
      70 |     };
    > 71 |     getDetailsSpy.and.returnValue(of(detailDataWithHistogram));
         |                       ^
      72 |     component.osd = { tree: { id: 0 } };
      73 |     component.refresh();
      74 |     expect(getDetailsSpy).toHaveBeenCalled();

```

Reply:
Thanks for the tip!

Reply:
I was referring to the blank link below the import of `of`  :D